core/domain/draft_upgrade_services_test.py

```diff
@@ -183,7 +221,34 @@ def test_convert_states_v32_dict_to_v33_dict(self):
                         'value': False
                     }
                 }
-            }).to_dict())
+            }),
+            exp_domain.ExplorationChange({
+                'cmd': exp_domain.CMD_EDIT_STATE_PROPERTY,
+                'state_name': 'state2',
+                'property_name': 'widget_customization_args',
+                'new_value': {
+                    'choices': {
+                        'value': [
+                            '<p>1</p>',
+                            '<p>2</p>',
+                            '<p>3</p>',
+                            '<p>4</p>'
+                        ]
+                    },
+                    'maxAllowableSelectionCount': {
+                        'value': 1
+                    },
+                    'minAllowableSelectionCount': {
+                        'value': 1
+                    }
+                }
+            })]
+        expected_draft_change_list = self.helper_to_test_schema_verison_updates(
```

Comment:
Here and elsewhere I think you should rename `expected_draft_change_list` to `migrated_draft_change_list`,   because  when `draft_change_list_v32` is migrated the expected dict is `draft_change_list_v33`.

Reply:
I would also add that "expected" values are usually hardcoded. You would then use logic to calculate the actual values and compare them with the expected ones.

Reply:
Thanks for the suggestion @donosco98, I changed it everywhere.